genet/core.py

```diff
@@ -2609,17 +2609,27 @@ def check_connectivity_for_mode(self, mode):
             )
         return con_desc
 
-    def generate_standard_outputs(self, output_dir, gtfs_day="19700101", include_shp_files=False):
+    def generate_standard_outputs(
```

Comment:
Why is it an option to produce geojson/shapefile in _addition_ to geoparquet rather than _instead of_?

Reply:
It started with just geojson, and then some transport people we were working with wanted shapefiles. Because we still preferred geojson, we just wanted an option to also output shapefiles if necessary. But I agree, it's better to have a single option and just call on a method twice with different options if you need other types